Sources/App/configure.swift

```diff
@@ -29,13 +29,17 @@ public func configure(_ config: inout Config, _ env: inout Environment, _ servic
     services.register(middlewares)
     
     config.prefer(MemoryKeyedCache.self, for: KeyedCache.self)
-    // Configure a SQLite database
-    let sqlite = try SQLiteDatabase(storage: .memory)
+
+    // Configure a PostgreSQL database
+    let config = PostgreSQLDatabaseConfig(hostname: "localhost", port: 5432, username: "oneleif", database: "website", password: "sudo", transport: .cleartext)
```

Comment:
I think we should load the credentials from environment variables, so we can quickly change the credentials without having to update the application code.

```swift
let config = PostgreSQLDatabaseConfig(
  hostname: Environment.get("DB_HOST") ?? "localhost", 
  port: Int(Environment.get("DB_PORT") ?? "") ?? 5432, 
  username: Environment.get("DB_USER") ?? "oneleif", 
  database: Environment.get("DB_NAME") ?? "oneleif-\(env.name)", 
  password: Environment.get("DB_PASSWORD") ?? "root", 
  transport: .cleartext
)
```